### PR TITLE
[1LP][RFR] Dockerbot use sprout checkout

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
this tries to use the sprout checkout command in dockerbot

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 41db086e608364a91c810890941f1baa8949fadb:

`cfme/scripting/sprout.py`:
- :bangbang: Line 27:61: **E999** *SyntaxError: invalid syntax*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: scripts/dockerbot/pytestbase/setup.sh [hunk 1]
```diff
@@ -216,7 +219,16 @@ run_n_log "find $CFME_REPO_DIR -name \"*.pyc\" -exec rm -rf {} \;"
 
 set +e
 
+if [ "$USE_SPROUT" = "yes" ];
+then
+    miq sprout checkout --populate-yaml  2>&1 >> $ARTIFACTOR_DIR/setup.txt &
```
Comment (psav) [inline scripts/dockerbot/pytestbase/setup.sh]: The miq sprout command has its own provision timeout, is the bit below to fail hard after 5 seconds? That isn't enough time for it to spin up and appliance if it doens't have one in the pool. I think we should let it run until the provisioning timer runs out.

Comment (RonnyPfannschmidt) [reply scripts/dockerbot/pytestbase/setup.sh]: it uses the do or die with 5 attempts each 10 second sleep, we can up that if necessary - for now it worked fine

to me its not clear what a good time-out would be
for the python implementation i'd opt for allowing the checkout command to fail/exit, and the waiting for the data to break out only if the command exits with a failed checkout

Comment (psav) [reply scripts/dockerbot/pytestbase/setup.sh]: right, I think we push it up for a 5 min max

File: scripts/dockerbot/pytestbase/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM fedora:23
 RUN dnf install -y gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel git nano python-devel gnupg gnupg2 libcurl-devel redhat-rpm-config findutils libffi-devel openssl-devel tesseract freetype-devel gcc-c++
-RUN git clone https://github.com/RedHatQE/cfme_tests.git
+RUN git clone https://github.com/RedHatQE/cfme_tests.gitt
```
Comment (mshriver) [inline scripts/dockerbot/pytestbase/Dockerfile]: typo here, extra t

File: scripts/dockerbot/pytestbase/setup.sh (hunk 1 above)
Comment (RonnyPfannschmidt) [reply scripts/dockerbot/pytestbase/setup.sh]: it was bumped from 5 10 to 5 60

**Pull request merged**